configuration/config-txt/boot.md

```diff
@@ -1,5 +1,26 @@
 ## Boot options in config.txt 
 
+### start_file, fixup_file
+
+These options specify the firmware files transfered to the Videocore GPU prior to boot.
+
+`start_file` specifies the Videocore (VC4) firmware file to use.
+`fixup_file` specifies the file used to fixup memory locations used in the `start_file` to match the GPU memory split. Note the the `start_file` and the `fixup_file` are linked - you cannot mix them.
```

Comment:
I would say that they must be a matched pair.

I also think it is worth saying that these are advanced options, and referring users to start_x and start_debug instead.